src/sync/pull.rs

```diff
@@ -261,10 +261,10 @@ pub async fn maybe_end_try_pull(
 #[derive(Debug, Default, PartialEq, Serialize)]
 #[cfg_attr(test, derive(Clone))]
 pub struct PullRequest {
-    #[serde(rename = "clientViewAuth")]
-    pub client_view_auth: String,
-    #[serde(rename = "clientViewURL")]
-    pub client_view_url: String,
+    #[serde(rename = "pullAuth")]
```

Comment:
It's only trypull that needs these fields: they tell repc where to send the pull request and what auth to use. Since the pull endpoint is not proxying, it doesn't need any further auth or url. I can remove these when I rebase.